docs/usecases/external_collaboration.rst

```diff
@@ -0,0 +1,364 @@
+
+
+      # build a pipeline with a Logistic Regression
+      model = make_pipeline(StandardScaler(), LogisticRegression())
+      # evaluate the model using cross-validation
+      cv_result = cross_validate(model, data, target, cv=3)
+      print(f'average model accuracy is {cv_result["test_score"].mean():.3f}')
+
+   Finally, the trained model is used to predict the penguin species of your own samples, and writes it to a file.
+
+   .. code-block::
+
+      # fit the model on data; predict Species of own data points
+      model.fit(data, target)
+      res = model.predict(local_samples)
+      local_samples['prediction'] = res
+      # save the prediction
+      local_samples.to_csv(args.output)
+
+
+To make things easier for the data provider, you can add a software container that includes all required software, using the ``datalad-containers`` extension.
+
+.. runrecord:: _examples/remote-analysis-105
+   :language: console
+   :workdir: usecases/remote-analysis/penguin-jelly
+
+   $ datalad containers-add software --url shub://adswa/resources:2
+
+Let's run the script on mock data in the software container.
+The results will not be correct because the data is only simulated, but it generates a run record that the data provider can very easily rerun - sparing you the need for detailed instructions in an Email or a README file.
+Keep in mind to properly define inputs and outputs, so that the analysis can easily be rerun.
+
+.. runrecord:: _examples/remote-analysis-106
+   :language: console
+   :workdir: usecases/remote-analysis/penguin-jelly
+
+   $ datalad containers-run -n software -m "run analysis on mock data" -i 'inputs/*/*table*.csv' -o "predictions.csv" "python3 code/predict.py"
+
+Note how bad the model performs - this accuracy will likely improve on the real data.
+We can tag the dataset state to make rerunning easier:
+
+.. runrecord:: _examples/remote-analysis-107
+   :language: console
+   :workdir: usecases/remote-analysis/penguin-jelly
+
+   $ git tag runme
+
+The question is: How does this look like on the real data?
+A data provider would clone your dataset and replace the subdataset with the actual sensitive dataset.
+
+We can pretend to do this:
+
+.. runrecord:: _examples/remote-analysis-108
+   :language: console
+   :workdir: usecases/remote-analysis/penguin-jelly
+
+   $ datalad drop --what datasets inputs
+   $ datalad clone -d . https://hub.datalad.org/edu/penguins.git inputs
+
+
+Then, we rerun the analysis:
+
+.. runrecord:: _examples/remote-analysis-109
+   :language: console
+   :workdir: usecases/remote-analysis/penguin-jelly
+
+   $ datalad rerun runme
+
+The data providers perspective
+""""""""""""""""""""""""""""""
+
+How does it look like from the data provider perspective?
+A possible first step could be the initial creation of a DataLad dataset.
+To be able to keep all or selected file contents private, data providers should make sure that
```

Comment:
OK, this is about preparing the dataset that has already been consumed in the section immediately above.

I find this confusing, because it jumps through time.

Maybe it would be better to split the "toy example" into three sections that are ordered chronologically:

1. data provider: prepare a dataset for remote analysis
2. data consumer: prepare a remote analysis
3. data provider: execute a remote analysis